app/views/groups/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<section class="d-flex flex-column cont">
+  <div class="log-in d-flex justify-content-between align-items-center">
+    <%= image_tag "bars-solid.svg", class: "icon"  %>
+    <li class="headers"> <h2 class= "headers"> CATEGORIES</h2></li>
+    <%= image_tag "magnifying-glass-solid.svg", class: "icon"  %>                 
+  </div>
+  
+  <% @groups.each do |group| %>
+    <%= link_to " /groups/#{group.id}/purchases", class: "text-decoration-none" do %>
+    <div class="card-custom my-2 p-2">
+      <div class="card-body">
+        <div class="d-flex align-items-center">
+          <% if group.icon.attached? %>
+            <div>
+              <%= image_tag url_for(group.icon), class: "img" %>
+            </div>
+          <% end %>
+          <div class="d-flex flex-column ms-3">
+            <div class="d-flex justify-content-between align-items-center w-100 gap-2">
```

Comment:
- Kindly improve the design, Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.
- [x] Kindly follow the provided design (width, font, size, height, background, color...)
1. The `ADD NEW CATEGORY` button must always be at the bottom of your page. ⬇️  
2. Add this white border around the ADD NEW CATEGORY button text. 👍🏻 
3. The `ADD NEW PURCHASE` button has a similar problem. Kindly fix it as well. 💯  👍🏻
4. The total amount number should be bold and the text should be normal, Please increase this font width. 👍🏻
5. The card should have a category image, name, `created date`, and amount. kindly let's display the create date for each category card. 🛀🏻 
6. Please respect the height properties for each category or group card.
7. I don't think it is necessary to have two hamburger menu buttons, Kindly remove one 🚮 

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/234975771-04da4afc-af65-4a7f-aab2-6e01a5009c5a.png)| ![image](https://user-images.githubusercontent.com/79058364/219786288-a43c8eb4-d309-4b09-9b4e-02b929ccd7fb.png)